content/docs/user-guide/gems/reference/_index.md

```diff
@@ -123,7 +123,6 @@ toc: true
 | [Certificate Manager](./network/certificate-manager) | The Certificate Manager Gem provides access to authentication files for secure game connections from Amazon S3, files on disk, and other 3rd party sources. |
 | [Http Requestor](./network/http-requestor) | The HTTP Requestor Gem provides functionality to make asynchronous HTTP/HTTPS requests and return data through a user-provided call back function. |
 | [Metastream](./network/twitch/metastream) | The Metastream Gem provides functionality for an HTTP server that allows broadcasters to customize game streams with overlays of statistics and event data from a game session. |
-| [Multiplayer](./network/multiplayer) | The Multiplayer Gem provides support for synchronizing O3DE components across a network in a fault-tolerant way, allowing for the development of multiplayer games. |
```

Comment:
Did you mean to remove this reference? Don't we want a reference to the Multiplayer Gem here?

Reply:
If you look up a bit, it's in the Multiplayer section right above Networking. It seemed redundant to have it in both.